tests/src/core/testqgsfield.cpp

```diff
@@ -336,6 +351,13 @@ void TestQgsField::equality()
   field2.setSplitPolicy( Qgis::FieldDomainSplitPolicy::GeometryRatio );
   QVERIFY( field1 == field2 );
 
+  field1.setMetadata( {{ static_cast< int >( Qgis::FieldMetadataProperty::GeometryCrs ), QStringLiteral( "abc" )}, {2, 5 }} );
```

Comment:
I find this not very readable. Does it need to be in the ::isNumeric test ?
Are you basically calling .setMetadata (map) here ? Is it needed for the purpose of the test, compared to just `.setMetadata(GeometryCrs, "abc")`

Reply:
It's in the equality test -- it's so we can test whether multiple metadata items correctly impact the equality operator.